### PR TITLE
Pause docs: page backgrounds vertices and quads

### PR DESCRIPTION
**Pull request by Dragorn421**
The *thrilling* continuation to pause docs split into smaller PRs, this time documenting the geometry generation and drawing for the background of each page

See #1366 for the "mother PR" with full pause docs

### CODE REVIEW

File: src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c [hunk 1]
```diff
@@ -13,167 +13,485 @@
...
+    gPauseSave24Tex,
 };
 
 #if OOT_NTSC
-#define EQUIPMENT_TEXS(language) ((language) != LANGUAGE_JPN ? sEquipmentENGTexs : sEquipmentJPNTexs)
-#define SELECT_ITEM_TEXS(language) ((language) != LANGUAGE_JPN ? sSelectItemENGTexs : sSelectItemJPNTexs)
-#define MAP_TEXS(language) ((language) != LANGUAGE_JPN ? sMapENGTexs : sMapJPNTexs)
-#define QUEST_STATUS_TEXS(language) ((language) != LANGUAGE_JPN ? sQuestStatusENGTexs : sQuestStatusJPNTexs)
-#define SAVE_TEXS(language) ((language) != LANGUAGE_JPN ? sSaveENGTexs : sSaveJPNTexs)
+#define EQUIPMENT_TEXS(language) ((language) != LANGUAGE_JPN ? sEquipPageBgQuadsENGTexs : sEquipPageBgQuadsJPNTexs)
```
Comment (krm01) [inline src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c:454]: i think this was mentioned before by someone else in another PR, i can't remember.. i also can't remember what was decided - but - the abbreviation `Texs` is such a weird one, i would be happy to just use `TEXTURES`/`Textures`

disregard if the conclusion to that other discussion is that `texs` is acceptable (sorry I tried looking but i can't find it)

File: src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c [hunk 2]
```diff
@@ -13,167 +13,485 @@
...
+    gPauseSelectItem13Tex,
+    gPauseSelectItem14Tex,
+    // column 3
+    gPauseSelectItem20JPNTex,
+    gPauseSelectItem21Tex,
+    gPauseSelectItem22Tex,
+    gPauseSelectItem23Tex,
+    gPauseSelectItem24Tex,
+};
+static void* sMapPageBgQuadsJPNTexs[] = {
```
Comment (fig02) [inline src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c:59]: I like line breaks between all set of data, despite the grouping you have going on here. i think the comments still suffice for separating groups

**Review by fig02 — APPROVED** [review]

File: src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c [hunk 3]
```diff
@@ -1850,113 +2172,117 @@ s16 func_80823A0C(PlayState* play, Vtx* vtx, s16 arg2, s16 arg3) {
...
-    s16 phi_a1;
-    s16 phi_a2;
-    s16 phi_t3;
-    s16 phi_t1;
+    s16 bufIAfterPageSections;
+    s16 pageBgQuadX;
+    s16 pageBgQuadY;
+    s16 i;
+    s16 j;
+    s16 bufI;
```
Comment (krm01) [inline src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c:2180]: not a big fan of `bufI` but after looking at the usage.... guh. i can't think of anything meaningfully better to suggest

File: src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c (hunk 1 above)
Comment (fig02) [reply src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c:454]: I still also do not like texs, but they are already prominent in lots of places in the codebase and should be all changed at once if we decide to change it. I dont mind it propagating a little more cause its still easy to find and fix

dragorn is a texs enjoyer, also

Comment (krm01) [reply src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c:454]: womp womp. ok thanks!

File: src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c (hunk 3 above)
Comment (Dragorn421) [reply src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c:2180]: it's the next free index in the vertex buffer :^)

**Review by krm01 — APPROVED** [review]

**Comment by cadmic** [comment]
No opinion on the PR, but could you update disassembly data please?

File: src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c (hunk 2 above)
Comment (Dragorn421) [reply src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c:59]: I will add the line returns

**Review by Dragorn421 — COMMENTED** [review]
TODO:
- line returns
- sync disassembly function names

**Comment by Dragorn421** [comment]
TODO done

**Pull request merged**